Should fix #2900 I think

jmarrec opened the pull request:
ftH20 (zero) instead of ftH2O (capital letter) for the new pump fields

jmarrec commented:
It didn't... I'm trying various things on the IDD before I need to resort to debugging the QuantityRegex.cpp > regexUnit() function. Rebuild time in between is long...

jmarrec commented:
Ok, I no longer get the crash reported on #2900 now that I have modified the IDD.

It still worries me a lot that the "1/ftH_{2}O" is getting parsed as "1/ftH_{2}O^2" though... not sure what to do here. I'll pick up in the AM.

jmarrec commented:
![pasted image at 2017_12_14 06_16 pm](https://user-images.githubusercontent.com/5479063/34005819-6efc2882-e0fc-11e7-8c10-4b9aeb576c08.png)

jmarrec commented:
Oops I committed my changes to QuantityRegex.i, you should discard that.

macumber commented on openstudiocore/src/model/ModelCore.i:
```diff
@@ -168,7 +168,7 @@ namespace model {
     return os.str();
   }
 
-  // This really shouldn't be necessary
```
@jmarrec you the grammar police now? ;-)

jmarrec replied:
I replaced “shouldn’t” by “should not” because the ‘ was breaking syntax highlighting for the rest of the file on my vim editor :)

macumber commented:
Thanks for cranking on this @jmarrec !  I'll run the tests and see if anything breaks before merging

jmarrec commented:
I tested all openstudio_tests, all tests ok (aside from FT ScheduleInterval #2885), the EMS ones (fixed in #2903), and 2 core/path tests that always fail on Linux (see slack)

I launched OpenStudio-resources tests to make sure (it really shouldn't affect it...), and a few failed, but I don't think they have anything to do with this

jmarrec commented:
Added missing autosize-related methods to fix https://github.com/NREL/OpenStudio-resources/issues/22:

* new terminals: AirTerminal:DualDuct:VAV:OutdoorAir, AirTerminal:DualDuct:ConstantVolume, and,
* CoolingTower:SingleSpeed and TwoSpeed: added for design range temperature and design approach temperature: these aren't being reported to the SQL file, so instead it does what E+ does and goes to the attached loop's Sizing:Plant object (see [CondenserLoopTowers.cc](https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/CondenserLoopTowers.cc#L2480) around this line)

macumber commented on openstudiocore/src/model/People.cpp:
```diff
@@ -423,9 +423,9 @@ namespace detail {
+    OptionalDouble temp = peopleDefinition().spaceFloorAreaperPerson();
     if (temp) {
-      return 1.0 / temp.get();
+      return temp.get() * multiplier();
```
This should get divided by the load instance multiplier right?

macumber replied:
@jmarrec if you have 100ft2/person and then you add a multiplier of 5, you want 100ft/(5person) right?  Or do you want 500ft2/person (larger multiplier gets fewer people).  Multipliers are kind of weird with this one

macumber replied:
That might be why this was the way it was?

jmarrec replied:
I agree with you. The space should be 5 times more crowded... We should check if the multiplier is strictly greater than 0 before dividing too.

jmarrec replied:
The way it was is wrong though.

peopleDefintion.setSpaceFloorAreaperPerson(10) 
then from the linked people: 
people.spaceFloorAreaPerPerson => I will get a non initialized currently!
